src/network/event_loop.rs

```diff
@@ -510,4 +565,20 @@ impl EventLoop {
 			},
 		}
 	}
+
+	fn reset_relay_reservation(&mut self) {
+		self.relay_reservation = RelayReservation {
+			id: PeerId::random(),
```

Comment:
Should the ID here be random or empty?

Reply:
this is just for resetting it to something different other than current ID

I need to check, not sure but I think that PeerId doesn't implement that
will check and fix it, if possible